utils/scripts/ocm/ocm.py

```diff
@@ -1186,11 +1233,56 @@ def compare_with_old_version_file(self):
     subparsers = parser.add_subparsers(
         title="Available sub commands", help="Available sub commands"
     )
+    # Argument of update_ocm_policy
+    update_ocm_policy = subparsers.add_parser(
+        "update_ocm_policy",
+        help="Parser to update_ocm_channel",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    update_ocm_policy.add_argument(
+        "--update-policies-json",
+        help="pass the json file to update ocm policy",
```

Comment:
Could you give a example of sample json file here.

Reply:
Okay, then dont write it here. But, add a sample json file in the downstream repository for reference.

Reply:
sure